src/test/scala/edu/ie3/simona/model/grid/RefSystemSpec.scala

```diff
@@ -6,43 +6,56 @@
 
 package edu.ie3.simona.model.grid
 
-import org.scalatest.flatspec.AnyFlatSpec
+import edu.ie3.simona.test.common.UnitSpec
 import org.scalatest.matchers.should.Matchers
-import squants.{Dimensionless, Each}
-import squants.electro.{Amperes, ElectricPotential, Kilovolts, Ohms}
-import squants.energy.{Megawatts, Power, Kilowatts}
+import squants.{Dimensionless, Each, ElectricCurrent}
+import squants.electro.{
+  Amperes,
+  ElectricPotential,
+  ElectricalResistance,
+  Kilovolts,
+  Ohms,
+}
```

Comment:
These are a lot of imports, so it may make sense to import the overaching class completely?  => `squants.electro.*`